src/Components/Auth/ForgotPassword.js

```diff
@@ -0,0 +1,226 @@
+            > Get code
+            </Button>
+          </FormGroup>
+          <FormGroup>
+            <Input
+              type="text"
+              name="code"
+              placeholder="verification code"
+              disabled={!mailSent}
+              value={values.code}
+              onChange={(e) => {
+                handleChange(e);
+              }}
+            />
+            {errors.code && (
+              <p style={{ "text-align": "left" }}>{errors.code}</p>
+            )}
+          </FormGroup>
+          <FormGroup>
+          <div class="d-flex bd-highlight align-items-center">
+            <Input
+              type={visibility.password ? "text" : "password"}
+              name="password"
+              placeholder="new password"
+              value={values.password}
+              disabled={!mailSent}
+              onChange={(e) => {
+                handleChange(e);
+              }}
+            />
+          <span onClick={() => {toggleVisibility("password")}} class={visibility.password?"fa fa-fw fa-eye-slash field_icon toggle-password m-n4":"fa fa-fw fa-eye field_icon toggle-password m-n4"}></span>
+            </div>
+            {errors.password && (
+              <p style={{ "text-align": "left" }}>{errors.password}</p>
+            )}
+          </FormGroup>
+          <FormGroup>
+          <div class="d-flex bd-highlight align-items-center">
+            <Input
+              type={visibility.confirm_password ? "text" : "password"}
+              name="confirm_password"
+              placeholder="confirm password"
+              value={values.confirm_password}
+              disabled={!mailSent}
+              onChange={(e) => {
+                handleChange(e);
+              }}
+            />
+          <span onClick={() => {toggleVisibility("confirm_password")}} class={visibility.confirm_password?"fa fa-fw fa-eye-slash field_icon toggle-password m-n4":"fa fa-fw fa-eye field_icon toggle-password m-n4"}></span>
+            </div>
+            {errors.confirm_password && (
+              <p style={{ "text-align": "left" }}>{errors.confirm_password}</p>
+            )}
+          </FormGroup>
+
+          <Button
+            color="primary"
+            style={{ "margin-bottom": "10px" }}
+            size="sm"
+            block
+            type="submit"
+            disabled={!mailSent}
+            onClick={()=>{toggle()}}
+            color="success"
+          >
+              {loggingIn ? (
+                <div class="container">
+                  <Spinner class="align-right"size="sm" color="light"/>
+                </div>
+              ):"Update password"}
+            </Button>
+            <Button
+              color="primary"
+              style={{ "margin-bottom": "10px" }}
+              size="sm"
+              block
+              onClick={()=>props.login()}
+              color="primary"
+            >
+            Log In
+            </Button>
+        </Form>
```

Comment:
![image](https://user-images.githubusercontent.com/39180629/121423280-b63db080-c98d-11eb-836d-7e79f5b6861f.png)
First thing this verification code form should be rendered conditionally after the user has pressed the get code button and after a successful verification code sent notification.
Second, after pressing the get code button the verification code form submit button goes into the loading state.